add changelog (#700)
run-ai/docs: docs/home/changelog/hotfixes-2-15.md

## Version 2.15.4 - January 5, 2024

### Fixed issues

| Internal ID | Description |
|--|--|
| RUN-15026 | Fixed an issue in workloads that were built on a cluster that does not support the NFS field. |
| RUN-14907 | Fixed an issue after an upgrade where the *Analytics* dashboard was missing the time ranges from before the upgrade. |
| RUN-14903 | Fixed an issue where internal operations were exposed to the customer audit log. |
| RUN-14062 | Fixed an issue in the *Overview* dashboard where the content for the *Running Workload per Type* panel did not fit. |

## Version 2.15.2 - February 5, 2024

### Fixed issues

| Internal ID | Description |
|--|--|
| RUN-14434 | Fixed an issue where the *Allocated GPUs* metric was multiplied by seven. |

## Version 2.15.1 - December 17, 2023

### Release content

* <!-- RUN-14077 - [runai-cli] allow configuring client burst and QPS -->Added environment variables for customizable QPS and burst support.

* <!-- RUN-13968 - Prometheus high availability -- allow changing # of replicas -->Added the ability to support running multiple Prometheus replicas.

### Fixed issues

| Internal ID | Description |
|--|--|

> | RUN-14292 | Fixed an issue where Bright installations were failing due to missing `create cluster` permissions. |

Reviewer: Bright needs to be replaced with BCM

> | RUN-14292 | Fixed an issue where Bright installations were failing due to missing `create cluster` permissions. |

Author: ```suggestion
| RUN-14292 | Fixed an issue where BCM installations were failing due to missing `create cluster` permissions. |
```

| RUN-14289 | Fixed an issue where metrics were not working due to an incorrect parameter in the cluster-config file. |
| RUN-14198 | Fixed an issue in services where multi nodepool jobs were not scheduled due to an unassigned nodepool status. |
| RUN-14191 | Fixed an issue where a consolidation failure would cause unnecessary evictions. |

> | RUN-14154 | Fixed an issue where the cluster wizard dropdown listed versions that were incompatible with the installed control plane. |

Reviewer: "cluster wizard" is actually "new cluster" something (wizard?)

> | RUN-14154 | Fixed an issue where the cluster wizard dropdown listed versions that were incompatible with the installed control plane. |

Author: ```suggestion
| RUN-14154 | Fixed an issue in the *New cluster* form, whefre the dropdown listed versions that were incompatible with the installed control plane. |
```

> | RUN-13956 | Fixed an issue where templates were not edited successfully. |

Reviewer: Jobs templates

> | RUN-13956 | Fixed an issue where templates were not edited successfully. |

Author: ```suggestion
| RUN-13956 | Fixed an issue where the *Jobs* templates were not edited successfully. |
```

---
title: Changelog Version 2.15
summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch version.
authors:
    - Jason Novich
date: 2024-Feb-25
---

The following is a list of the known and fixed issues for Run:ai V2.15.

## Version 2.15.9 - February 5, 2024

### Fixed issues

| Internal ID | Description |
|--|--|
| RUN-15296 | Fixed an issue where the `resources` field was deprecated when creating a project. Now use `nodePoolResources. |

## Version 2.15.4 - January 5, 2024

### Fixed issues

| Internal ID | Description |
|--|--|
| RUN-15026 | Fixed an issue where building workloads to send to the cluster were not built inside volumes for backwards compatibility. |
| RUN-14907 | Fixed an issue where after an upgrade, the Analytics Dashboard is missing the time ranges from before the upgrade.|

> | RUN-14903 | Fixed an issue where the project status is audited creating long logs. |

Reviewer: I would remove it. Does not seem customer facing + KCL issues are mapped usually by our team

> | RUN-14062 | Fixed an issue where the Running Workload per Type Graph box content is too big for the frame. |

Reviewer: Where? In the overview dashboard. Missing the context

## Version 2.15.2 - February 5, 2024

### Fixed issues

| Internal ID | Description |
|--|--|
| RUN-14645 | Fixed an issue where a workload volume is mounted as read-only when created in the workspace training form. |

> | RUN-14635 | Fixed an issue where the Status updater was missing the correct permissions. |

Reviewer: That's the solution, which customers care less about if at all. What did this issue cause? You should ask Roman, as there is no info in the bug.

> | RUN-14472 | Fixed an issue where template updates were not being applied to the workload. |

Reviewer: Not clear which template and what was the impact on the user.

> | RUN-14434 | Fixed an issue where `runai_allocated_gpu_count_per_gpu` was multiplied by seven. |

Reviewer: Seems like it dud not happen all the time. When does it happen? The details of the inaccuracy are less relevant. What does runai_allocated_gpu_count_per_gpu mean for a customer?

> | RUN-14425 | Fixed an issue when calculating deserved quota metrics for project, nodepool, or cluster causing unlimited quotas to display -1. |

Reviewer: So the symptom is that an unlimited project quota is displayed like -1 in some cases. What are the cases? I did not understand the relevance on mentioning nodepool & cluster

> | RUN-13401 | Fixed an issue where there was no accurate error information when a predicate fails (was `not found`). |

Reviewer: what can a customer do with this info? Did you check if we have the term predicate documented??
